config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-presubmits.yaml

```diff
@@ -0,0 +1,33 @@
+presubmits:
+  kubernetes/cloud-provider-vsphere:
+  - name: pull-cloud-provider-vsphere-test
+    agent: kubernetes
+    context: pull-cloud-provider-vsphere-test
+    branches:
+    - master
+    always_run: true
+    rerun_command: "/test pull-cloud-provider-vsphere-test"
+    trigger: "(?m)^/test( all| pull-cloud-provider-vsphere-test),?(\\s+|$)"
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180716-9145034c9-master
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=execute"
+        - "--timeout=600"
```

Comment:
this flag is in minutes, 600 minutes seems excessive for a unit test

Reply:
you are right, all our in-tree provider unit tests are very fast, this out-of-tree unit tests will be similar, i reduce it but still keep it at several minutes level (6m) since we are adding more unit tests.

Reply:
sure, I just wouldn't want someone stuck waiting for 600 minutes on a hung job.